src/app/pages/connect-page/connection-details/connection-details.page.ts

```diff
@@ -2,6 +2,7 @@ import {Component, OnInit} from "@angular/core";
 import {ActivatedRoute} from "@angular/router";
 import { Router} from "@angular/router";
 import { ConnectModelService } from '../_services/connect.model.service';
+import { AlertService } from "../_services/alert/alert.service";
```

Comment:
You'll want to import the original alert.service, rather than making a copy of it.